src/lib/Preview.js

```diff
@@ -90,8 +97,10 @@ class Preview extends EventEmitter {
     /** @property {AssetLoader[]} - List of asset loaders */
     loaders = loaderList;
 
-    /** @property {Logger} - Logger instance */
-    logger;
+    /** @property {FileMetrics} - File metrics tracker instance */
+    fileMetrics;
+
+    /** @property {Logger} - Logging system instance */
```

Comment:
Can we not add the variable here as well? Seems kinda weird to have the doc here but then init it in the constructor